packages/rancher-cis-benchmark/charts/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: {{ template "cis.namespace" . }}
+  name: cis-operator-serviceaccount
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: {{ template "cis.namespace" . }}
+  labels:
+    app.kubernetes.io/name: rancher-cis-benchmark
+    app.kubernetes.io/instance: release-name
+  name: cis-serviceaccount
```

Comment:
what is this serviceaccount used for? I see it is referenced by clusterrolebinding but not actuallly used by any of pods.

Reply:
It is being used by the pods that get launched by the operator for every CIS scan created by user

Reply:
https://github.com/rancher/cis-operator/blob/master/pkg/securityscan/job/job.go#L72